Docs/how_to_build_on_linux.md

```diff
@@ -1,136 +1,194 @@
-<h1>How to build CARLA on Linux</h1>
----
+<h1>Linux build</h1>
 
-!!! note
-    CARLA requires Ubuntu 16.04 or later.
 
-Install the build tools and dependencies
+* [Requirements](#requirements):  
+	* __System specifics:__ Minimum to run CARLA. 
+	* __Dependencies:__ Needed to build the project. 
+* [GitHub](#github):   
+	* __Create a GitHub account:__ Provides access to repositories. 
+	* __Install git:__ Enables terminal commands. 
+	* __Link GitHub and Unreal Engine:__ Allows access to private Unreal Engine repositories.
+* [Unreal Engine](#unreal-engine): 
+	* __Download UE4.22:__ Specific version to run CARLA. 
+	* __Build UE4.22:__ Needed to build the project. 
+* [CARLA build](#carla-build): 
+	* __Clone repository:__ Get the CARLA project in a local computer. 
+	* __Get assets:__ Download the assets, which are stored separately. 
+	* __Set the environment variable:__ Ensures communication between CARLA and Unreal Engine.
+	* __make CARLA:__ Build the server, client and packages. 
+
+The build process can be quite long and tedious. This documentation tries to make things clear, but the CARLA forum is open to anybody who wants to post unexpected issues, doubts or suggestions. There is a specific section for installation issues in Linux. Feel free to login and become part of the community. 
+
+<div class="build-buttons">
+<!-- Latest release button -->
+<p>
+<a href="https://forum.carla.org/" target="_blank" class="btn btn-neutral" title="Go to the latest CARLA release">
+CARLA forum</a>
+</p>
+</div>
+
+---------------
+##Requirements
+
+<h4>System specifics</h4>
+* __Ubuntu 16.04 or later:__ migration to Ubuntu 18 is slowly coming forward, but today's date there is still support for 16.04. 
```

Comment:
Weird sentence. What about:

Currently migrating to Ubuntu 18.04, but version 16.04 is still being supported.